File: Java/Strings/pg0002.java
//Palindrome check – Verify if a string reads the same forward and backward.

import java.util.*;

public class pg0002 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String s1 = sc.nextLine();
        String s2 = sc.nextLine();
        String rev = "";
        for(int i = s1.length() - 1;i>=0;i--){
            rev += s1.charAt(i);
        }
        if(s2.equals(rev)){
            System.out.println("Palindrome");
        }
        else{
            System.out.println("Not a Palindrome");
        }
    }
}

File: Python/Strings/pg0002.py
#Palindrome check – Verify if a string reads the same forward and backward.


s1 = input()
s2 = input()
rev = ""
for i in s1:
    rev = i+ rev
if s2 == rev:
    print("Palindrome")
else:
    print("Not a Palindrome")
